bbc/simorgh
# Legacy media support in MediaLoader

Related to https://jira.dev.bbc.co.uk/browse/WSTEAM1-980

Overall changes
======
- Adds new `legacyMedia` configuration for MediaLoader to parse out a valid SMP config from legacy content
- Adds `legacyMedia` check to the MediaArticlePage component

Testing
======
- Visit http://localhost.bbc.com:7080/arabic/multimedia/2013/12/131208_iraq_blast_?renderer_env=test
- Confirm the media content is visible and is playable, which it previously wasn't

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
